### ISSUE TITLE
r831: Executing w_do_call mfc42 fails to find files

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noahocker/winetricks#280]
```
C.f http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=691935

If this is fixed in later SVN commit, please let me know.

Command 280 of 7 $winetricks mfc42
Executing w_do_call mfc42
Executing load_mfc42
Executing mkdir -p /home/marco/.cache/winetricks/vcrun6
Downloading
http://download.microsoft.com/download/vc60pro/update/1/w9xnt4/en-us/vc6redistse
tup_enu.exe
to /home/marco/.cache/winetricks/vcrun6
--2012-10-31 15:03:19--
http://download.microsoft.com/download/vc60pro/update/1/w9xnt4/en-us/vc6redistse
tup_enu.exe
Risoluzione di download.microsoft.com (download.microsoft.com)...
64.214.206.35, 64.214.206.10
Connessione a download.microsoft.com
(download.microsoft.com)|64.214.206.35|:80... connesso.
Richiesta HTTP inviata, in attesa di risposta... 404 Not Found
2012-10-31 15:03:20 ERRORE 404: Not Found.

------------------------------------------------------
Downloading
http://download.microsoft.com/download/vc60pro/update/1/w9xnt4/en-us/vc6redistse
tup_enu.exe
failed
```

Original issue reported on code.google.com by `jari.aalto.fi@gmail.com` on 1 Nov 2012 at 4:30

**Comment by GoogleCodeExporter** [comment]
```
Hi, an you confirm if this is fixed in latest SVN?
C.f http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=691935

r960 2013-06-12 austinenglish@gmail.com winetricks: version bump
r959 2013-06-12 austinenglish@gmail.com winetricks: remove stray debug code
r958 2013-06-11 daniel.r.kegel One more try at adapting to wine-1.6-rc1
r957 2013-06-10 daniel.r.kegel bump version to 20130609 (really, did it in last 
commit, oops)
r956 2013-06-10 daniel.r.kegel flash: update
r955 2013-06-10 daniel.r.kegel Handle version 1.6-rc1
r954 2013-05-21 austinenglish@gmail.com dotnet20: bug 30162 is fixed in 1.5.31
r953 2013-05-20 austinenglish@gmail.com winetricks: bump copyright years
r952 2013-05-05 austinenglish@gmail.com winetricks: also set windowscodecsext 
to native when using native windowscodecs
r951 2013-05-02 austinenglish@gmail.com winetricks: update dotnet11 sha1sum
```

Original comment by `jari.aalto.fi@gmail.com` on 13 Jun 2013 at 9:08

**Comment by GoogleCodeExporter** [comment]
```
This was fixed in r866.
```

Original comment by `austinenglish@gmail.com` on 15 Jun 2013 at 8:56
- Changed state: **Fixed**
